db/schema.rb

```diff
@@ -33,6 +33,39 @@
   add_index "authentications", ["uid", "provider"], :name => "index_authentications_on_uid_and_provider", :unique => true
   add_index "authentications", ["user_id", "provider"], :name => "index_authentications_on_user_id_and_provider", :unique => true
 
+  create_table "c_rater_feedback_items", :force => true do |t|
```

Comment:
Hmm, I was thinking about too. But there is also another option.

We will have to create a new model related to the argumentation block submission. It will obviously reference C-Rater feedback items of the open response questions. When it comes to multiple choice answer and feedback, there are two options:

1) we could create generic feedback item model as per your suggestion
2) we could just store multiple choice answer and feedback explicitly in this new model (so simply add a few columns there, like `main_question_answer`, `main_question_feedback`, etc.)

Separate model makes sense for C-Rater, as it can be useful for debugging. Each feedback item corresponds to one HTTP request and HTTP metadata is also stored there. 

When it comes to multiple choice question feedback, the whole operation would be as simple as copying current answer text and feedback text. It's not too risky and additional feedback item model doesn't bring too many benefits. Except from the fact that it would be similar to C-Rater feedback approach. We also don't need all the columns (status, score, response_info etc.), so we would need to either make them more generic or use another db table.